Source/Scene/Scene.js

```diff
@@ -1848,6 +1848,9 @@ define([
         var environmentState = scene._environmentState;
         var renderPass = frameState.passes.render;
         environmentState.skyBoxCommand = (renderPass && defined(scene.skyBox)) ? scene.skyBox.update(frameState) : undefined;
+        if (defined(scene.skyAtmosphere)) {
```

Comment:
Make sure to check that the globe is defined as well, otherwise the line below will throw an error.